src/components/admin/AdminCard.tsx

```diff
@@ -73,6 +74,7 @@ const AdminCard = (props: AdminCardProps) => {
           Active?
         </Text>
         <Switch onChange={handleActiveChange} mt={2.5} ml={3} isChecked={isChecked} />
+		<DeleteIcon ml={5} mt={2} onClick={() => handleDelete(assignmentOrLocation.id)} />

```

Comment:
Render out the `DeleteIcon` and call the `handleDelete` method (passed in as a prop0 with the ID.